pkg/processor/runtime/shell/test/shell_test.go

```diff
@@ -43,7 +43,11 @@ func (suite *TestSuite) TestOutputs() {
 	statusOK := http.StatusOK
 	statusInternalError := http.StatusInternalServerError
 
-	headersContentTypeTextPlain := map[string]string{"content-type": "text/plain; charset=utf-8"}
+	headersContentTypeTextPlain := map[string]string{
```

Comment:
Not sure I understand why this change is warranted. Seems like this is only meant to be the content-type header.